www/js/survey/enketo/enketo-add-note-button.js

```diff
@@ -42,23 +30,37 @@ angular.module('emission.survey.enketo.add-note-button',
 
   const updateLabel = () => {
     const localeCode = $translate.use();
-    if ($scope.notesConfig?.['filled-in-label'] && trip.tripAddition?.length > 0) {
+    if ($scope.notesConfig?.['filled-in-label'] && timelineEntry.additions?.length > 0) {
       $scope.displayLabel = $scope.notesConfig?.['filled-in-label']?.[localeCode];
     } else {
       $scope.displayLabel = $scope.notesConfig?.['not-filled-in-label']?.[localeCode];
     }
   }
   $scope.$watch('notesConfig', updateLabel);
-  $scope.$watch('trip.tripAddition', updateLabel);
+  $scope.$watch('timelineEntry.additions', updateLabel);
 
+  // return a dictionary of fields we want to prefill, using start/enter and end/exit times
   $scope.getPrefillTimes = () => {
-    const timeBounds = $scope.timeBounds();
-    const begin = timeBounds.start_fmt_time || timeBounds.enter_fmt_time;
-    const stop = timeBounds.end_fmt_time || timeBounds.exit_fmt_time;
+    const begin = $scope.timelineEntry.start_fmt_time || $scope.timelineEntry.enter_fmt_time;
+    const stop = $scope.timelineEntry.end_fmt_time || $scope.timelineEntry.exit_fmt_time;
+
+    const momentBegin = moment.parseZone(begin);
+    let momentStop;
```

Comment:
```suggestion
    let momentStop = moment.parseZone(stop);
```

In implementation this causes this result: 
<img width="371" alt="image" src="https://user-images.githubusercontent.com/61334340/224567076-aaea8cb6-290a-49d3-9b40-93f4c331441a.png">

Where all the trips and place additions work properly, except the last place addition which prefills the stop_time as 1 hour after the start time. Does this sound correct?

Reply:
Please see the comment in the code right below this

In real-life usage, the last place will represent the place you're currently at. We will prefill with the current time

However, for dev testing we just load old test trips, and the current time can seem inappropriate. We could end up with an end time that is earlier than the start time, which would be invalid

So, I implemented a quick check for whether the start time is the same as today. If not, we just use start time + 1 hour.

Reply:
I see the comment and the reasoning makes sense.

However in testing I still do need to have the above recommended change, as the add-notes-button will not work without `let momentStop = moment.parseZone(stop);`, since the return statement uses `momentStop.format('HH:mm:ss.SSSZ')` which will not be able to function if `momentStop` is not set to a value.

Reply:
Ah, yes you're right. Your suggested change is the intended behavior.
I misunderstood before, when I was looking at this on my phone